src/gui/render.cpp

```diff
@@ -51,6 +51,22 @@
 Render_t render;
 ScalerLineHandler_t RENDER_DrawLine;
 
+const char* to_string(const PixelFormat pf)
+{
+	switch (pf) {
+	case PixelFormat::Indexed8: return "Indexed8";
+	case PixelFormat::BGR555: return "BGR555";
+	case PixelFormat::BGR565: return "BGR565";
+	case PixelFormat::BGR888: return "BGR888";
+	case PixelFormat::BGRX8888: return "BGRX8888";
+	default: assertm(false, "Invalid pixel format"); return {};
+	}
+}
+
+uint8_t get_bits_per_pixel(const PixelFormat pf) {
+	return enum_val(pf);
```

Comment:
I'm not sure if essentially casting the enum this is a good idea. What if we have to introduce `RGB888` in addition to `BGR888` in the future?

Reply:
Then we'll rethink the approach 😄 Unlikely, though.

But yeah I could remove the value assignments from the enums and just do another switch/case block here. I'm like 50/50, it's doesn't matter much for now.